mesh_aws_client/mesh_mailbox.py

```diff
@@ -0,0 +1,136 @@
+"""Mailbox class that handles all the complexity of talking to MESH API"""
```

Comment:
This entire file seems to be duplicated from mesh_common.py

Reply:
Oops, that wasn't supposed to be committed. Never mind, it's going to be the getting rid of mesh_client stuff here.